pom.xml

```diff
@@ -314,32 +309,31 @@
     <version.com.github.tomakehurst.wiremock>1.53</version.com.github.tomakehurst.wiremock>
     <version.com.google.android>4.1.1.4</version.com.google.android>
     <version.com.google.testing.compile>0.11</version.com.google.testing.compile>
-    <version.jakarta-regexp>1.4</version.jakarta-regexp>
-    <version.jakarta.json.bind>1.0.2</version.jakarta.json.bind>
     <version.org.antlr4>4.8</version.org.antlr4>
-    <version.org.apache.cxf>3.2.14</version.org.apache.cxf>
+    <!-- CXF's version is aligned with Jetty -->
+    <version.org.apache.cxf>3.3.9</version.org.apache.cxf>
     <version.org.apache.camel>2.24.0</version.org.apache.camel>
-    <version.org.codehaus.cargo>1.7.11</version.org.codehaus.cargo>
-    <version.org.freemarker>2.3.30</version.org.freemarker>
+    <version.org.codehaus.cargo>1.9.3</version.org.codehaus.cargo>
+    <version.org.freemarker>2.3.28</version.org.freemarker>
```

Comment:
why is here the freemarker downgrade? the previous version is a CVE fix was due https://app.snyk.io/vuln/SNYK-JAVA-ORGFREEMARKER-1076795